### PR TITLE
Campaign entities

### PR DESCRIPTION
**Pull request by aepton**
Breaking out campaign-related entities into separate PR from campaign finance filing entities.

### CODE REVIEW

File: proposals/drafts/campaign_entities.rst [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+id
+    Open Civic Data-style id in the format ``ocd-cf-election/{{uuid}}``
+
+date
+    Date of the Election.
+
+jurisdiction
+    Jurisdiction with at least one race occurring on this Election date.
+
+is_primary
```
Comment (palewire) [inline proposals/drafts/campaign_entities.rst:81]: I suspect we might need more than just an ``is_primary`` flag. There are also special elections when people resign or die in office, as well recall elections, to consider. A "type" field might be appropriate here as well.

File: proposals/drafts/campaign_entities.rst [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+This is not necessarily a Person: for instance, "yes" or "no" to a ballot
+measure.
+
+id
+    Open Civic Data-style id in the format ``ocd-cf-candidate/{{uuid}}``
+
+election
+    Election this Candidate is contesting.
+
+party
```
Comment (palewire) [inline proposals/drafts/campaign_entities.rst:54]: Does OCD already have a Party model?

**Comment by fgregg** [comment]
With due respect to @aepton , I'm pretty interested in seeing if we could try to closely follow the [VIP election specification](https://github.com/votinginfoproject/vip-specification). It would be great to have a higher degree of interoperability to that existing data standard. Would also really like input from the openelex people. paging @djbridges and @ghing

**Comment by dwillis** [comment]
I agree with @fgregg - the VIP standard is one that is well-known and growing in use. In terms of parties, things get a little hairy in some states, where candidates run on multiple party lines (CT & NY are especially fun).

File: proposals/drafts/campaign_entities.rst (hunk 1 above)
Comment (jsfenfen) [reply proposals/drafts/campaign_entities.rst:81]: Also, runoff elections. This is super helpful to know in fed-land, where hard money is capped at $2,700 (2016)  *per election*, so the total max allowed individual contribution goes up when there's a runoff.

Comment (aepton) [reply proposals/drafts/campaign_entities.rst:81]: Yep, this should be an enumerated type. Fixing.

**Comment by aepton** [comment]
Oh wow, yeah, we should totally just use VIP for this. Closing.

**Pull request closed without merging**